src/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -45,8 +49,11 @@
       <Link>Common\System\IO\Compression\ZipTestHelper.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
`netstandard` configuration means you run on netstandard. Adding the parameter is fine. I forgot to ask why you have to add a netcoreapp configuration here?